### PR TITLE
Website category names changed

### PR DESCRIPTION
**Pull request by comradekingu**
Happy to see the localization effort :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -27,19 +27,19 @@ <h2>Open source re-implementation of RollerCoaster Tycoon 2</h2>
       </li>
       <li>
         <a href="https://openrct2.org/downloads/latest/develop">Download Development Build</a>
-        <div class="information">Windows, Linux &amp; OS X</div>
+        <div class="information">Linux, Windows and macOS</div>
```
Comment (comradekingu) [inline index.html]:
```suggestion
        <div class="information">Linux, Android, Windows and macOS</div>
```

**Comment by duncanspumpkin** [comment]
Android support is quite a bit more patchy compared to all the other platforms. Should it maybe be the last item on the list? Why did you change the order of the platforms? The largest proportion of players are Windows. It's not alphabetical either which would also be fine.

**Comment by comradekingu** [comment]
@duncanspumpkin I sorted by happiness.

**Comment by Gymnasiast** [comment]
> @duncanspumpkin I sorted by happiness.

We are not a cult. Please just sort them by number of users, with Android at the end: "Windows, macOS, Linux & Android".

As for the other three changes: I don't think there is a good reason for them, so I'd like to see them reversed, unless you can convince me otherwise.

**Comment by Gymnasiast** [comment]
No reply from OP in 10 days, closing this.

**Pull request closed without merging**